web/client/utils/FeatureGridUtils.js

```diff
@@ -162,19 +162,50 @@ module.exports = {
     applyAllChanges: (orig, changes = {}) => applyChanges(orig, changes[orig.id] || {}),
     applyChanges,
     gridUpdateToQueryUpdate: ({attribute, operator, value, type} = {}, oldFilterObj = {}) => {
+        const andGroup = {
+            id: 9999, // dummy number representing group id
+            logic: "AND",
+            index: 0
+        };
+
+        let mergeFilters = oldFilterObj.groupFields 
+            ? (oldFilterObj.groupFields.filter(obj => obj.id === 1 ).length > 0 && oldFilterObj.filterFields.length > 0 
```

Comment:
1.filter objects coming from query panel have group id of 1. if there are at least more than one group fields with id of 1 and at least more than one filterfields then query panel filter is active